client/my-sites/upgrades/domain-management/style.scss

```diff
@@ -25,27 +25,37 @@
 	border-top: 1px solid $gray-light;
 	margin: 16px -16px -16px -16px;
 	padding: 16px;
+	display: flex;
+	justify-content: flex-end;
+	flex-direction: row;
```

Comment:
We probably don't need this specific rule since the default value of `flex-direction` is `row`.